kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue

```diff
@@ -75,22 +74,46 @@
         v-model="confirmedPassword"
       />
 
-      <ui-select
-        :name="$tr('selectFacility')"
-        :placeholder="$tr('selectFacility')"
-        :label="$tr('facility')"
-        :value="selectedFacility"
-        :options="facilityList"
-        :invalid="facilityIsInvalid"
-        :error="facilityIsInvalidText"
-        @blur="facilityBlurred = true"
-        @input="updateSelection"
-      />
+      <!-- ToS modal only shows up if the box empty prior to clicking -->
+      <!-- Using UI Checkbox for blur events and v-model -->
+      <label for="terms-agreement-checkbox">
+        <button
+          type="button"
+          class="terms-agreement-view-prompt"
+          @click="showTerms = true"
+        >
+          {{ $tr('viewTermsOfServicePrompt') }}
+        </button>
+
+        <k-checkbox
+          id="terms-agreement-checkbox"
+          :class="['terms-agreement-checkbox', termsNotAgreed ? 'invalid' : '']"
+          :checked="termsAgreed"
+          @change="termsAgreed = $event"
+          @blur="termsAgreementCheckboxBlurred = true"
+          :label="$tr('termsAgreementLabel')"
+        />
+
+        <span
+          v-if="termsNotAgreed"
+          class="terms-agreement-error-box"
+        >
+          {{ termsNotAgreedText }}
+        </span>
+      </label>
 
       <k-button :disabled="busy" :primary="true" :text="$tr('finish')" type="submit" />
 
     </form>
 
+    <core-modal
+      v-if="showTerms"
+      @cancel="showTerms = false"
+      :title="$tr('termsOfServiceModalHeader')"
+    >
+      <iframe class="terms" src="/content/databases/tos.txt"></iframe>
```

Comment:
Let's add `frameborder="0"` to get rid of the iframe border?

Reply:
HTML4 only ^ used css to do it instead

Reply:
Oops, I will have to do the same to the About iframes too then.